htdocs/api/v0.0.3-dev/candidates/Visit.php

```diff
@@ -211,13 +252,14 @@ public function handlePUT()
      * @param integer $subprojectID The subproject for the new visit
      * @param string  $VL           The visit label of the visit to
      *                              be created
+     * @param integer $CID          The ID of the center of the visit
      *
      * @return none
      */
-    function createNew($CandID, $subprojectID, $VL)
+    function createNew($CandID, $subprojectID, $VL, $CID)
     {
         try {
-            \TimePoint::isValidVisitLabel($CandID, $subprojectID, $VL);
+            \TimePoint::isValidVisitLabel($CandID, $subprojectID, $VL, $CID);
```

Comment:
need to pass $CID to `\Timepoint::createNew(...)` below

Reply:
done